db/attachment.go

```diff
@@ -253,6 +279,21 @@ func (db *Database) ForEachStubAttachment(body Body, minRevpos int, callback Att
 	return nil
 }
 
+func GetAttVersion(meta map[string]interface{}) int {
+	if ver, ok := meta["ver"].(json.Number); ok {
+		if version, err := ver.Int64(); err != nil {
+			return AttVersion1
+		} else {
+			return int(version)
+		}
+	} else if ver, ok := meta["ver"].(int); ok {
```

Comment:
What scenarios exist that set this value as an `int`?

Do we want to be made aware (via logging or an error) of invalid types rather than silently return version 1?

Reply:
This value is set as int from the unit test cases, when looking at this further, it make sense to use the base.ToInt64 while   obtaining the attachment version from meta.